test/spec/adapters/rubicon_spec.js

```diff
@@ -326,6 +326,59 @@ describe('the rubicon adapter', () => {
 
         });
 
+        it('should send digitrust params', () => {
+          window.DigiTrust = {
+            getUser: function() {}
+          };
+          sandbox.stub(window.DigiTrust, 'getUser', () =>
+            ({
+              success: true,
+              identity: {
+                privacy: {optout: false},
+                id: 'testId',
+                keyv: 'testKeyV'
+              }
+            })
+          );
+
+          rubiconAdapter.callBids(bidderRequest);
+
+          let request = xhr.requests[0];
+
+          let query = request.url.split('?')[1];
+          query = parseQuery(query);
+
+          let expectedQuery = {
+            'dt.id': 'testId',
+            'dt.keyv': 'testKeyV'
+          };
+
+          // test that all values above are both present and correct
+          Object.keys(expectedQuery).forEach(key => {
+            let value = expectedQuery[key];
+            expect(query[key]).to.equal(value);
+          });
+
+          delete window.DigiTrust;
+        });
+
+        it('should not send digitrust params', () => {
```

Comment:
thinking that rather than testing that the params don't exist when window.DigiTrust doesn't exist, this would be a more useful test if DigiTrust existed but the `optout` flag was `true` or maybe `success` was `false`